src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -2084,7 +2087,9 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Security\SecureString.Unix.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Threading\LowLevelMonitor.Unix.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Threading\TimerQueue.Unix.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)System\TimeZoneInfo.Unix.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\TimeZoneInfo.AnyUnix.cs" />
```

Comment:
I don't really get the difference between "AnyUnix" and "Unix". Can we come up with some better names?

Reply:
Do you have any suggestions?  .AnyUnix is similar to .AnyOS in intent.

Reply:
I think the separation between platforms is as such
.............TimeZoneInfo
............../.................\ 
........Win32............Non-Win32 [originally Unix] (AnyUnix)
............................./..................................\ 
..........Non-Android (Unix)..................Android

Not sure what is a preferred naming to convey this.

Reply:
I was thinking `.Unix`, `.Android`, and `.NonAndroid`. I think that declares the most intent up front.

> .AnyUnix is similar to .AnyOS in intent.

Yes, but we don't have `OS` when we have `AnyOS`. That's the issue - which one means "all Unix"? "Unix" or "AnyUnix"? They both mean the same thing to me.

Reply:
Renamed to those (`.Unix`, `.Unix.Android`, `.Unix.NonAndroid`)